corehq/apps/api/resources/v0_5.py

```diff
@@ -26,6 +29,8 @@
 from corehq.apps.userreports.models import ReportConfiguration
 from corehq.apps.users.models import CommCareUser, WebUser, Permissions
 from corehq.util import get_document_or_404
+from corehq.apps.users.models import CommCareUser, WebUser, Permissions, CouchUser
```

Comment:
F811 redefinition of unused 'CommCareUser' from line 30
F811 redefinition of unused 'Permissions' from line 30
F811 redefinition of unused 'WebUser' from line 30